MultiOracle.md

```diff
@@ -0,0 +1,450 @@
+possible while still containing `[start - minFail, end + minFail]` by repeatedly cutting them in half on either side.
+This process results in 1-3 secondary oracle CETs for ever single CET of the primary oracle. 
+
+### 2-of-2 Oracles with Bounded Error
+
+In the two oracle case, mark one oracle as primary and the other as secondary.
+First, generate all adaptor points for the primary oracle as is done in the single-oracle
+case for [numeric outcome DLCs](NumericOutcome.md).
+Then we shall construct a list of 1-3 secondary points with which to combine each primary point.
+This scheme can be thought of as computing a set of 1-3 CETs for the secondary oracle to cover
+at least the minimum range and at most the maximum range of outcomes allowed given the each
+primary oracle's CET.
+
+The maximum possible support range is parameterized by a maximum error exponent, `maxErrorExp`,
+where all difference more than `maxError = 2^maxErrorExp` are guaranteed **not** to be supported.
+
+The minimum supported range is parameterized by a minimum failure exponent, `minFailExp`, where
+all differences of up to `minFail = 2^minFailExp` are guaranteed to be supported.
+It is required that `minFailExp` be strictly less than `maxErrorExp` because this gap is key to the 
+existence of the following algorithm.
+
+If the two oracles sign outcomes that are between `minFail` and `maxError` apart, then this procedure
+can provide only probabilistic guarantees.
+That said, DLC participants can decide in advance whether they wish to maximize the probability of
+failure or of success in these cases.
+
+With these parameters in mind, we shall now detail the algorithm for computing the secondary oracle's
+adaptor points given those of the first.
+
+#### Algorithm
+
+First, let us define some subprocedures.
+
+* `computeCETIntervalBinary` takes as input a list of digits corresponding to a CET, and the total
+  number of binary digits to be signed by an oracle, and returns an interval `[start, end]`
+  which that CET covers.
+* `numToVec` takes as input an integer representing the left endpoint of a CET's range, the number
+  of binary digits to be signed by the oracle, and a number of digits to ignore, and returns
+  the binary digits corresponding to that CET.
+
+We take as input the number of digits to be signed by the oracle, `numDigits`, the binary digits of the primary
+oracle's CET which we wish to cover with the secondary oracle, `cetDigits`, `maxErrorExp`, `minFailExp`,
+and a boolean flag `maximizeCoverage` which signals whether to maximize the probability of success for
+outcomes differing by less than `maxError` and more than `minFail` or not (in which case failure is maximized).
+
+Let `[start, end] = computeCETIntervalBinary(cetDigits, numDigits)`,
+let `halfMaxError = 2^(maxErrorExp - 1)`,
+and let `maxNum = (2^numDigits) - 1`.
+
+We then split into cases:
+
+TODO: Pictures here
+
+* **Case** Small CET (`end - start + 1 < maxError`)
```

Comment:
I think I mentioned already, but I'm not a big fan of this naming. It confuses me to use the term CET for what I think is outcome value. I see that of course there is a relation but it would IMHO be better to have distinct terms.